cmake/resources/generate_resources.cmake

```
@@ -88,6 +88,15 @@ if (WIN32)
     list(APPEND RES_AUTOGEN_FILES "${CMAKE_BINARY_DIR}/autogen/windows.rc")
 endif ()
 
+set(WINDOWS_ARGUMENTS "@Invalid()") # empty QVariant() in QSettings
+if (CHATTERINO_USE_GDI_FONTENGINE AND Qt${MAJOR_QT_VERSION}_VERSION GREATER_EQUAL "6.8.0")
```

```suggestion
if (CHATTERINO_USE_GDI_FONTENGINE AND Qt${MAJOR_QT_VERSION}_VERSION VERSION_GREATER_EQUAL "6.8.0")
```

@Nerixyz  can you confirm this works for you, and then apply this change? see https://cmake.org/cmake/help/v3.7/command/if.html#command:if - iirc 6.8.1 is GREATER_EQUAL 6.8.0, and 6.10.0 is not GREATER_EQUAL 6.8.0

---

Oh right i missed the version 🤦